runtime/manta/src/lib.rs

```diff
@@ -516,6 +516,9 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type ChannelInfo = ParachainSystem;
 	type VersionWrapper = PolkadotXcm;
 	type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
+	type ControllerOrigin = CollatorSelectionUpdateOrigin;
+	type ControllerOriginConverter = XcmOriginToTransactDispatchOrigin;
+	type WeightInfo = ();
```

Comment:
We should benchmark `cumulus_pallet_xcmp_queue` in next release.

Reply:
Log it please.

Reply:
File a issue #537